tests/test_hook_implementations.py

```diff
@@ -0,0 +1,81 @@
+import os
+import sys
+import shutil
+
+import pytest
+
+
+try:
+    import ecl2df.hook_implementations.jobs
+    from ert_shared.plugins.plugin_manager import ErtPluginManager
+except ImportError:
+    pytest.skip(
+        "ERT is not installed, or Python2. Skipping hook implementations.",
+        allow_module_level=True,
+    )
```

Comment:
Same as the other file :slightly_smiling_face:

Reply:
Fixed.